pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -478,6 +500,14 @@ def __init__(self, interim=None, center=None, vel_scale=_DEFAULT_CARTESIAN_VEL_S
         self._interim = interim
         self._center = center
 
+    def __eq__(self, other):
+        if isinstance(other, Circ):
+            return super(Circ, self).__eq__(other) \
+                   and self._interim == other._interim \
+                   and self._center == other._center
+
+        return NotImplemented
```

Comment:
```suggestion
        else:
            return NotImplemented
```
makes it more readable

Reply:
I do not really agree, that adding extra stuff to read makes it more readable. Id rather just delete the empty line.

Reply:
Is this with the new code still an issue?